app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemViewModel.kt

```diff
@@ -165,6 +165,7 @@ class VaultItemViewModel @Inject constructor(
             is VaultItemAction.Common.ConfirmCloneWithoutFido2CredentialClick -> {
                 handleConfirmCloneClick()
             }
+            is VaultItemAction.Common.RestoreVaultItemClick -> handleRestoreItemClicked()
```

Comment:
Newline. lol

Reply:
I guess is the convention we are following if the above branch has `{ ... }` then the next one should follow a `/n`?

Reply:
Yes, the formatter should do this for you as well